tests/Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot

```diff
@@ -23,5 +23,9 @@ Launch Jupyterhub via Routes
 Launch Jupyterhub via App
    Click Element  xpath://header[@id='page-main-header']/div[2]/div[1]/div[1]/nav/button
    Sleep  1
-   Click Element  xpath://a[contains(@href, 'https://rhods-dashboard')]
-   Switch Window  NEW
\ No newline at end of file
+   #Click Element  xpath://a[contains(@href, 'https://rhods-dashboard')]
+   #Switch Window  NEW
+
+   # Temporary workaround, open new tab and go to RHODS dashboard
+   Execute Javascript    window.open('${ODH_DASHBOARD_URL}')
```

Comment:
why not using `Go To`?

Reply:
`Go To` doesn't open a new tab, it uses the current window